pinot-core/src/main/java/org/apache/pinot/core/operator/filter/EmptyFilterOperator.java

```diff
@@ -26,10 +29,9 @@
  * Singleton class which extends {@link BaseFilterOperator} that is empty, i.e. does not match any document.
  */
 public final class EmptyFilterOperator extends BaseFilterOperator {
-  private EmptyFilterOperator() {
```

Comment:
Don't remove this private constructor. We want to use this operator as a singleton

Reply:
Fixed.